jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1296,7 +1296,7 @@ properties:
             fits_hdu: SCI
             blend_table: True
           time:
-            title: UTC time of position and velocity vectors in ephemeris (MJD)
+            title: [d] MJD time of position and velocity vectors
```

Comment:
This entry needs to be enclosed in quotes, to prevent the yaml parser from choking on the [d] entry.